app/models/pool/invite.rb

```diff
@@ -27,6 +27,12 @@ def sent_to_candidate?
     sent_to_candidate_at.present?
   end
 
+  def matching_application_choice(form)
+    form.application_choices
```

Comment:
Since the invite has_one application _form, you don't need to pass the form. 
```suggestion
  def matching_application_choice
    application_form.application_choices
```